src/components/hooks/use-platform-query-param.tsx

```diff
@@ -1,11 +1,26 @@
+import { useEffect, useState } from 'react'
 import { useQueryParam, StringParam } from 'use-query-params'
 
 export const usePlatformQueryParam = () => {
     const [platform] = useQueryParam('platform', StringParam)
     const platforms = ['derivgo', 'p2p']
-    const is_deriv_go = platform === 'derivgo'
-    const is_deriv_p2p = platform === 'p2p'
-    const has_platform = platforms.includes(platform)
+    const [is_deriv_go, setIsDerivGo] = useState(true)
+    const [is_deriv_p2p, setIsDerivP2P] = useState(true)
+    const [has_platform, setHasPlatform] = useState(false)
+
+    // initialize value for the first render
+    useEffect(() => {
+        setIsDerivGo(platform === 'derivgo')
+        setIsDerivP2P(platform === 'p2p')
+        setHasPlatform(platforms.includes(platform))
+    }, [])
```

Comment:
@farhan-nurzi-deriv I think only the second `useEffect` should be enough as it will be called for the first time anyway even if `platforms` doesn't change. I'm not sure tho, Can you check on that please 🙏🏻

Reply:
Yeah the second useEffect is enough. Thank you @farzin-deriv 🙏 
cc: @habib-deriv